LeVent/LeVent.csproj

```diff
@@ -43,4 +43,8 @@
 	  </None>
 	</ItemGroup>
 
+	<ItemGroup>
```

Comment:
I don't think this section is required.  It will automatically be inluded due to the files.

Reply:
@cjdutoit Fixed